huaweicloud/services/apig/resource_huaweicloud_apig_instance.go

```diff
@@ -44,364 +61,386 @@ func ResourceApigInstanceV2() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			"region": {
-				Type:     schema.TypeString,
-				Optional: true,
-				Computed: true,
-				ForceNew: true,
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				ForceNew:    true,
+				Description: `The region in which to create the dedicated instance resource.`,
 			},
 			"name": {
 				Type:     schema.TypeString,
 				Required: true,
-				ValidateFunc: validation.StringMatch(regexp.MustCompile("^([\u4e00-\u9fa5A-Za-z][\u4e00-\u9fa5A-Za-z-_0-9]{2,63})$"),
-					"The name consists of 3 to 64 characters, starting with a letter. Only letters, digits, "+
-						"hyphens (-) and underscore (_) are allowed."),
+				ValidateFunc: validation.All(
+					validation.StringMatch(regexp.MustCompile("^([\u4e00-\u9fa5A-Za-z][\u4e00-\u9fa5A-Za-z-_0-9]*)$"),
+						"The name can only contain letters, digits, hyphens (-) and underscore (_), and must start "+
+							"with a letter."),
+					validation.StringLenBetween(3, 64),
+				),
+				Description: `The name of the dedicated instance.`,
 			},
 			"edition": {
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
 				ValidateFunc: validation.StringInSlice([]string{
-					"BASIC", "PROFESSIONAL", "ENTERPRISE", "PLATINUM",
+					string(EditionBasic),
+					string(EditionProfessional),
+					string(EditionEnterprise),
+					string(EditionPlatinum),
+					string(Ipv6EditionBasic),
+					string(Ipv6EditionProfessional),
+					string(Ipv6EditionEnterprise),
+					string(Ipv6EditionPlatinum),
 				}, false),
+				Description: `The edition of the dedicated instance.`,
 			},
 			"vpc_id": {
-				Type:     schema.TypeString,
-				Required: true,
-				ForceNew: true,
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `The ID of the VPC used to create the dedicated instance.`,
 			},
 			"subnet_id": {
-				Type:     schema.TypeString,
-				Required: true,
-				ForceNew: true,
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `The ID of the VPC subnet used to create the dedicated instance.`,
 			},
 			"security_group_id": {
-				Type:     schema.TypeString,
-				Required: true,
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `The ID of the security group to which the dedicated instance belongs to.`,
 			},
-			"available_zones": {
-				Type:     schema.TypeList,
-				Required: true,
-				Elem:     &schema.Schema{Type: schema.TypeString},
+			"availability_zones": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				ForceNew:    true,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Description: `schema: Required, The name list of availability zones for the dedicated instance.`,
```

Comment:
```suggestion
				Description: `schema: Required; The name list of availability zones for the dedicated instance.`,
```

please use `;` as the end of extra schema.